### ISSUE TITLE
Documentation: Example of 'env' in docker_service container_spec is incorrect

### ISSUE THREAD

**Issue by rolandcrosby** [terraform-providers/terraform-provider-docker#276]
The [Advanced example](https://www.terraform.io/docs/providers/docker/r/service.html#advanced) for the docker_service resource shows an `env` block, like this:

```
resource "docker_service" "foo" {
  name = "tftest-service-basic"

  task_spec {
    container_spec {
      image = "repo.mycompany.com:8080/foo-service:v1"

...

      env {
        MYFOO = "BAR"
      }
```

This is incorrect: that value is now a regular map (`env = {MYFOO = "BAR"}`) as you can see in [this test](https://github.com/terraform-providers/terraform-provider-docker/blob/master/docker/resource_docker_service_test.go#L245).

**Comment by GMSteuart** [comment]
pr #281

**Comment by mavogel** [comment]
Closed by #281
